.goreleaser.yml

```diff
@@ -0,0 +1,132 @@
+before:
+  hooks:
+  - make js.build
+
+builds:
+  - main: ./cmd/ttn-lw-stack
+    binary: ttn-lw-stack
+    ldflags:
+    - -s
+    - -w
+    - -X go.thethings.network/{{.ProjectName}}/pkg/version.BuildDate={{.Date}}
```

Comment:
I don't think it's appropriate to use `{{.ProjectName}}` here. The go module path is `go.thethings.network/lorawan-stack/pkg/version`, which isn't necessarily the same as the name of the dir in which this repo is cloned.

Reply:
I'll set `project_name` explicitly then, since this may affect other artifacts as well

Reply:
Fine, but that doesn't change anything about the fact that the go module path is something different.

Reply:
How is that related here?

Reply:
`go.thethings.network/{{.ProjectName}}` in these lines must change to `go.thethings.network/lorawan-stack`.